package.json

```diff
@@ -454,6 +454,12 @@
                     "default": true,
                     "description": "Format inline documentation.",
                     "scope": "window"
+                },
+                "julia.format.kw": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Format kw.",
```

Comment:
@ZacLN What would be a good description? I'm not actually sure what this option does :)